media/CircleEditor/view.js

```diff
@@ -168,6 +168,10 @@ view.View = class {
             }
         }
         this._page = page;
+
+        if (typeof (this._host._viewingNode) === 'number') {
```

Comment:
How about following?
At first, I misunderstood that `_viewingNode` can be another type than `number` :)

```suggestion
        if (this._host._viewingNode !== null) {
```

Reply:
Thank you! I think it's a good idea!